lib/median/index.js

```diff
@@ -1,15 +1,14 @@
 import mean from '../mean'
 
-const median = (list) => {
-  var width = 2 - list.length % 2
+export default (list) => {
```

Comment:
let's revert the change for now

It will replace empty string with the function name in `SIZES.md`